index.html

```diff
@@ -10,28 +10,239 @@
     <title>TripHouse Project</title>
 </head>
 <body>
-<section class="offer-container">
-    <h3 class="offer-title">What do we offer</h3>
-    <div class="offer-title-line"></div>
-    <ul class="offer-blocks-wrapper">
-        <li class="block">
-            <img class="img-container" src="./img/project_images/support.png" alt="image:support">
-            <p class="block-info">Support 24/7</p>
-        </li>
-        <li class="block">
-            <img class="img-container" src="./img/project_images/communicate.png" alt="image:communicate">
-            <p class="block-info">Communicate directly</p>
-        </li>
-        <li class="block">
-            <img class="img-container" src="./img/project_images/book.png" alt="image:book-online">
-            <p class="block-info">Book online</p>
-        </li>
-        <li class="block">
-            <img class="img-container" src="./img/project_images/review.png" alt="image:guests-reviews">
-            <p class="block-info">Real guest reviews</p>
-        </li>
-    </ul>
-</section>
+<main>
+    <section class="main">
+        <div class="main-container">
+            <div class="main-header">
+                <img src="img/project_images/main-logo-icon.png" alt="" class="main-header-logo">
+                <ul class="main-header-list">
+                    <li class="main-header-items">Stays</li>
+                    <li class="main-header-items">Attractions</li>
+                    <li class="main-header-items night-icon"></li>
+                    <li class="main-header-items account-icon"></li>
+                </ul>
+            </div>
+            <h1 class="main-title">Discover stays<br>
+                to live, work or just relax</h1>
+            <div class="main-search">
+                <form action="/" method="POST" class="search-form">
+                    <label for="destination" class="label-search"><span class="label">Your destination or hotel
+                        name</span><br>
+                        <input id="destination" class="search-input input-place" type="text" name="place"
```

Comment:
How about to add `<datalist>` here? 
[Example](https://stackoverflow.com/questions/18796221/creating-a-select-box-with-a-search-option)

Reply:
Applied! Thanks)